style.css

```diff
@@ -94,9 +94,24 @@ ul { list-style-type: square; }
 
 .episode {
 	border-top: 1px dashed #d6611b;
-	margin-top: 2em;
-	padding-top: 2em;
+	margin-top: 1em;
+	padding-top: 1em;
 }
+	.episode_decor {
+		float: left;
+		padding-top: 0.25em;
+		padding-right: 0.5em;
```

Comment:
Looking at the screenshot again, I think we may want a bit more breathing room between the icon and the title. Otherwise I think we're ready to go!

Reply:
This is 1em:

![padding-right-1em](https://user-images.githubusercontent.com/36317762/81985306-f1cf4500-95ea-11ea-8ff7-95f604f202da.png)

Reply:
Perfect!